upgrading/upgrade-helm.adoc

```diff
@@ -17,8 +17,7 @@ If you have installed {product-title} by using Helm charts, to upgrade to the la
 
 [IMPORTANT]
 ====
-* In order to ensure optimal functionality it is recommended to keep your secured clusters up to date and on the same version as Central. Out of date Sensor versions are supported for 6 months, but may not have the full functionality of an up to date version.
-* Enabling automated upgrades for your secured clusters is strongly recommended. Should you prefer manually upgrading your secured clusters refer to the xref:upgrade-roxctl.adoc#upgrade-secured-clusters[manual on upgrading secured clusters].
+* In order to ensure optimal functionality it is recommended to keep your secured-cluster-services on the same version as central-services.
```

Comment:
Probably this would look better without the bulleted list decoration because there's just one item.

```suggestion
In order to ensure optimal functionality it is recommended to keep your secured-cluster-services on the same version as central-services.
```

Reply:
Fair point, done